cloudfoundry/resource_cf_app.go

```diff
@@ -24,10 +24,11 @@ import (
 // schema.BasicMapReader
 // DefaultAppTimeout - Timeout (in seconds) when pushing apps to CF
 const (
-	DefaultAppTimeout   = 60
-	DefaultBindTimeout  = 5 * time.Minute
-	DefaultStageTimeout = 15 * time.Minute
-	DefaultAppPort      = 8080
+	DefaultAppTimeout                   = 60
+	DefaultBindTimeout                  = 5 * time.Minute
+	DefaultStageTimeout                 = 15 * time.Minute
+	DefaultAppPort                      = 8080
+	DefaultBlueGreenPostStartupWaitTime = 10
```

Comment:
Maybe we should set it to 0 by default in order for it to not impact people who don't want it